modules/migrating-sccs-during-the-manual-upgrade.adoc

```diff
@@ -0,0 +1,360 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-roxctl.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-sccs-during-the-manual-upgrade_{context}"]
+= Migrating SCCs during the manual upgrade
+
+By migrating the security context constraints (SCCs) during the manual upgrade by using `roxctl` CLI, you can seamlessly transition the {rh-rhacs-first} services to use the {osp} SCCs, ensuring compatibility and optimal security configurations across Central and all secured clusters.
+
```

Comment:
could we add a sentence here? 

This step is required when upgrading to release 4.4 if you installed {product-title-short} for {osp} manually by using the manifest installation method.